📚 [DOC] Warn that autodoc expects rST docstrings, not Markdown. (#339)
executablebooks/MyST-Parser: docs/using/howto.md

(howto/autodoc)=
## Use `sphinx.ext.autodoc` in Markdown files

> The [Sphinx extension `autodoc`](sphinx:sphinx.ext.autodoc), which pulls in code documentation from docstrings, is currently hard-coded to output reStructuredText.

Reviewer: ```suggestion
The [Sphinx extension `autodoc`](sphinx:sphinx.ext.autodoc), which pulls in code documentation from docstrings, is currently hard-coded to parse reStructuredText.
```

> It therefore does not blend in well with MyST, which expects Markdown as input.

Reviewer: ```suggestion
It is therefore incompatible with MyST's Markdown parser.
```
